src/demo.js

```diff
@@ -283,87 +285,85 @@ export default class DemoTimeline extends Component {
     }
 
     return (
-      <Layout className="layout demo-layout">
-        <Layout.Content className="demo-layout-content">
-          <div style={{margin: 24}}>
-            <Form layout="inline">
-              <Form.Item label="No rows">
-                <InputNumber value={rows} onChange={e => this.setState({rows: e})} />
-              </Form.Item>
-              <Form.Item label="No items per row">
-                <InputNumber value={items_per_row} onChange={e => this.setState({items_per_row: e})} />
-              </Form.Item>
-              <Form.Item label="Snap (min)">
-                <InputNumber value={snap} onChange={e => this.setState({snap: e})} />
-              </Form.Item>
-              <Form.Item label="Date Range">
-                <DatePicker.RangePicker
-                  allowClear={false}
-                  value={rangeValue}
-                  showTime
-                  onChange={e => {
-                    this.setState({startDate: e[0], endDate: e[1]}, () => this.reRender());
-                  }}
-                />
-              </Form.Item>
-              <Form.Item>
-                <Button type="primary" onClick={() => this.reRender()}>
-                  Regenerate
-                </Button>
-              </Form.Item>
-              <Form.Item>
-                <Button onClick={this.zoomIn}>Zoom in</Button>
-              </Form.Item>
-              <Form.Item>
-                <Button onClick={this.zoomOut}>Zoom out</Button>
-              </Form.Item>
-              <Form.Item label="Custom Renderers">
-                <Switch onChange={this.toggleCustomRenderers} />
-              </Form.Item>
-              <Form.Item>
-                <Checkbox onChange={this.toggleSelectable} checked={selectable}>
-                  Enable selecting
-                </Checkbox>
-              </Form.Item>
-              <Form.Item>
-                <Checkbox onChange={this.toggleDraggable} checked={draggable}>
-                  Enable dragging
-                </Checkbox>
-              </Form.Item>
-              <Form.Item>
-                <Checkbox onChange={this.toggleResizable} checked={resizeable}>
-                  Enable resizing
-                </Checkbox>
-              </Form.Item>
-            </Form>
-            <div>
-              <span>Debug: </span>
-              {message}
-            </div>
+      <React.Fragment>
+        <div style={{margin: 24}}>
```

Comment:
Removed Layout + Layout.Content w/ React.Fragment, because it's width was "shrinked" in Storybook. Now the width is good. But we have a scrollbar issue. We always have 2 scrollbars. Typically appears when a div should have an unique child that "fills", but it has 2. And one has 100%. If the fix doesn't use the 2 css classes => they should be removed.